lib/LoaderRunner.js

```diff
@@ -253,7 +253,10 @@ exports.runLoaders = function runLoaders(options, callback) {
 	var contextDependencies = [];
 
 	// prepare loader objects
-	loaders = loaders.map(createLoaderObject);
+	// loaders = loaders.map(createLoaderObject);
+	for(var loader in loaders) {
+		loaders[loader] = createLoaderObject(loaders[loader]);
+	}
```

Comment:
Hey @sokra. I was responsible behind this change. Sorry for the mishap. It was a part of the change to remove `map` in favor of the regular `for loop`. It was my first contribution to the webpack community.